packages/collector/test/tracing/database/db2/test.js

```diff
@@ -882,10 +877,10 @@ mochaSuiteFn('tracing/db2', function () {
         .then(() =>
           testUtils.retry(() =>
             verifySpans(agentControls, controls, {
-              numberOfSpans: 14,
+              numberOfSpans: 15,
```

Comment:
I noticed there are now 15 spans, likely because I’m using the native `fs`, which may have caused the span count to increase. If the test continues to fail again, so I will investigate further.